### PR TITLE
Update docs for Algolia 2.1.5 changes.

### PR DESCRIPTION
**Pull request by acoumb**
## Description

_What did you add/update/change?_

## Type of suggestion

* [ ] Typo/grammar fix
* [ ] Updated outdated content
* [ ] New content
* [x] Updates related to a new version
* [ ] Other

## Product & version (if relevant)

Algolia 2.1.5

## Deadline (if relevant)

No deadline, when possible.

### CODE REVIEW

File: marketplace-and-integrations/integrations/algolia.md [hunk 1]
```diff
@@ -171,6 +171,23 @@ services.AddUmbraco(_env, _config)
     .Build();
 ```
 
+### Version 2.1.5 and up
```
Comment (alina-tincas) [inline marketplace-and-integrations/integrations/algolia.md:174]: move this section above ### Version 2.0.0 to 2.1.5, as the newest are documented first. So the order would be: Version 2.0.0 to 2.1.5 **then** Version 2.0.0 to 2.1.5 **then** Up to version 1.5.0

File: marketplace-and-integrations/integrations/algolia.md [hunk 2]
```diff
@@ -171,6 +171,23 @@ services.AddUmbraco(_env, _config)
...
+
+Following [this](https://github.com/umbraco/Umbraco.Cms.Integrations/issues/188) issue, that affects `Umbraco.TinyMCE` property editor in Umbraco 13, we have passed the `IProperty` object to the parse method of the converters.
+
+A custom converter will look like this, allowing developers to add their implementation based on the content property.
```
Comment (alina-tincas) [inline marketplace-and-integrations/integrations/algolia.md:178]:
```suggestion
As a resolution for [an issue](https://github.com/umbraco/Umbraco.Cms.Integrations/issues/188) that affects `Umbraco.TinyMCE` property editor in Umbraco 13, the `IProperty` object has been passed to the parse method of the converters.

A custom converter will look like this, allowing developers to add their implementation based on the content property:
```

**Review by alina-tincas — CHANGES_REQUESTED** [review]
Hi @acoumb thank you for the PR 🙌 

I have just few suggestions, can you please take a look into them? 😊

**Comment by acoumb** [comment]
Updated. Thanks @alina-tincas

**Review by alina-tincas — APPROVED** [review]

**Comment by alina-tincas** [comment]
Thank you as well for making the necessary changes 😊 #h5YR 

Merging this in 🙌

**Pull request merged**